NASA-AMMOS/aerie-ui
# Create and restore plan snapshots

resolves #876 

To test:
1. Create a plan with some activities
2. Click on the plan name on the nav bar to open the menu
3. Click on "Take Snapshot"
4. Fill in the snapshot name and click "Create Snapshot" button
5. Add more activities to the plan
6. Add the `snapshotId` search parameter with the snapshot ID number (e.g. `snapshotId=1`) to the url and hit enter
7. Verify that the correct activities are loaded associated with the snapshot (should be fewer activities)
